packages/docs/community/release-process/base-cli-contractkit-dappkit-utils.md

```diff
@@ -0,0 +1,81 @@
+  * Minor releases: approximately 8 times a year
+  * Patch releases: as needed
+
+Development builds will be identified as such: `x.y.z-dev`, and will be published as `x.y.z` when stable. 
+
+## Identifying releases
+
+### NPM
+
+You can find the npm packages in the following places:
+  * [@celo/base](https://www.npmjs.com/package/@celo/base)
+  * [@celo/celocli](https://www.npmjs.com/package/@celo/celocli)
+  * [@celo/contractkit](https://www.npmjs.com/package/@celo/contractkit)
+  * [@celo/dappkit](https://www.npmjs.com/package/@celo/dappkit)
+  * [@celo/utils](https://www.npmjs.com/package/@celo/utils)
+
+### Github tags
+
+To identify the commits included in a specific release and see which new features were added or bugs fixed, please refer to the [release notes](https://github.com/celo-org/celo-monorepo/releases) in the monorepo. Also to keep track of continual updates to the stable and dev versions of the packages, each package has a `CHANGELOG.md` file: [Base](https://github.com/celo-org/celo-monorepo/blob/master/packages/base/CHANGELOG.md), [Celocli](https://github.com/celo-org/celo-monorepo/blob/master/packages/cli/CHANGELOG.md), [Contractkit](https://github.com/celo-org/celo-monorepo/blob/master/packages/contractkit/CHANGELOG.md), [Dappkit](https://github.com/celo-org/celo-monorepo/blob/master/packages/dappkit/CHANGELOG.md), [Utils](https://github.com/celo-org/celo-monorepo/blob/master/packages/utils/CHANGELOG.md).
+All releases should be tagged with the version number, e.g. `contractkit-vX.Y.Z`. Each release should include a summary of the release contents, including links to pull requests and issues with detailed description of any notable changes. 
+
+### Communication
+
+The community will be notified of package updates through the following channels: 
+
+For all releases: 
+  * Each package’s `CHANGELOG.md` file, as mentioned above
+  * Github releases page, as mentioned above
+  * [Discord](https://discord.gg/6yWMkgM): #developer-chat, #mainnet, and #sdk
+
+For major releases: 
+  * Twitter: [@CeloDevs](https://twitter.com/CeloDevs)
+  * Mailing list: cLabs’ Tech Sync
+  * [Celo Forum](https://forum.celo.org/)
+
+
+
+## Build process
+
+The packages are published by following the instructions [here](https://github.com/celo-org/celo-monorepo/blob/master/README-dev.md).
+
+## Testing
+All builds of these packages are automatically tested for performance and backwards compatibility in CI. Any regressions in these tests should be considered a blocker for a release.
+Minor and major releases are expected to go through additional rounds of manual testing as needed to verify behavior under stress conditions.
+
+{% hint style="warning" %}
+Work in progress
+{% endhint %}
+
+## Promotion process 
+  * For a patch release: The first commit of this process should change the version number encoded in the source from `x.y.z` to `x.y.z+1-dev` and the final commit should change the published version number from `x.y.z-1`  to `x.y.z`.
```

Comment:
This is super tiny, but maybe change the second part to "the published version number from `x.y.z` to `x.y.z+1`." to keep it consistent with the first part and clear that those numbers correspond with one another? (Unless I'm mistaken here and this is supposed to indicate something else)

Reply:
Yah so this is a bit confusing, but basically what's in the repo right now is the current dev version which is one ahead of the current published version. Then,  when we publish a new version, it will take on what the current dev version number is , so the dev version must increment by 1. Let me know if this makes sense, it's a bit convoluted so an example might help!

Reply:
Yah so this is a bit confusing, but basically what's in the repo right now is the current dev version which is one ahead of the current published version. Then, when we publish a new version, it will take on what the current dev version number is , so the dev version must increment by 1. Let me know if this makes sense, it's a bit convoluted so i'm happy to walk through an example!

Reply:
Just checking if I understand:
- At the start of dev in the repo, we bump say `0.1.1 (x.y.z)` --> `0.1.2-dev`. At this point, current published version number is also `0.1.1 (x.y.z)`
- We publish a new version, bumping `0.1.1` --> `0.1.2` as well
Or is it this, which is how I would read it with the current numbering:
- At the start of dev in the repo, we bump say `0.1.1 (x.y.z)` --> `0.1.2-dev`. Current published version number is (`x.y.z-1 == 0.1.0`)
- We publish a new version, bumping `0.1.0` --> `0.1.1`

Thanks for the clarification!!

Reply:
Yes it's the second one! Minor change is that `0.1.1 (x.y.z)` -->` 0.1.2-dev` is actually `0.1.1 (x.y.z)-dev `--> `0.1.2-dev`, but I had forgotten the "-dev" in the PR so added it in a new commit!